### DISCUSSION TITLE
Open PDF files inside Ferdium?

### DISCUSSION THREAD

**Discussion by dev-allinappli** [ferdium/ferdium-app#1318] [Ideas]
We use Matrix/Element and often exchange PDF such as resumes. It would be useful to open them directly inside Ferdium like now in Thunderbird instead of downloading and open it after.

Would that help other people?

**Comment by GuillaumeSorel** [comment]
I see other people asking for such a feature https://github.com/orgs/ferdium/discussions/1432 & https://github.com/orgs/ferdium/discussions/1645. Any updates?

**Comment by NR2014** [comment]
I have same issue with Whatsapp .. its hard to download to folder and open it.. native app has easy opening straight to specific app , even web based whatsapp - browser open the file in respective app. 

Please make it open in respective software/app..
